src/components/course/data/tests/utils.test.jsx

```diff
@@ -18,3 +18,41 @@ describe('findCouponCodeForCourse', () => {
     expect(findCouponCodeForCourse(couponCodes)).toBeUndefined();
   });
 });
+
+describe('findEnterpriseOfferForCourse', () => {
+  const enterpriseOffers = [
+    {
+      enterpriseCatalogUuid: 'cats',
+      remainingBalance: 99,
+    },
+    {
+      enterpriseCatalogUuid: 'horses',
+      remainingBalance: 100,
+    },
+    {
+      enterpriseCatalogUuid: 'cats',
+      remainingBalance: 100,
+    },
+  ];
+
+  it('returns undefined if there is no course price', () => {
+    const catalogList = ['cats', 'bears'];
+    expect(findEnterpriseOfferForCourse({
+      enterpriseOffers, catalogList,
+    })).toBeUndefined();
+  });
+
+  it('returns undefined if there is no enterprise offer for the course', () => {
```

Comment:
nit: is this also true when the `coursePrice` is passed? At first glance, I thought this was potentially still asserting that `undefined` ii returned with no price.